06--Temple-of-DOM/README.md

```diff
@@ -0,0 +1,138 @@
+```
+
+### Journal, Week 1
+
+Continuing your journal entry in your `USERNAME.github.io`, start writing a Rough Draft based on your outline / stream of consciousness / brainstorming notes. Remember, this is _rough_... stay loose.
+
+### Githubbing to Know You, Part 2 (due Thu)
+
+You have your assignment. Here are your targets. Now you just have to _talk_ to someone.
+
+#### Hometown, :dog: or :cat:?
+
+| you ask      | they answer  |
+| ------------ | ------------ |
+| Epicurean306 | gatorpazz    |
+| JessyRiordan | pope410211   |
+| sjoyal       | Epicurean306 |
+| MStaehling   | JessyRiordan |
+| jorgehjr84   | kmattiko     |
+| pcreglow     | MStaehling   |
+| melrodbos    | gatorpazz    |
+| pope410211   | Epicurean306 |
+| kmattiko     | MStaehling   |
+| gatorpazz    | pope410211   |
+
+#### Faves :v: (only 2, only 2)
+
+| you ask      | they answer  |
+| ------------ | ------------ |
+| Epicurean306 | sjoyal       |
+| JessyRiordan | jorgehjr84   |
+| sjoyal       | JessyRiordan |
+| MStaehling   | JessyRiordan |
+| jorgehjr84   | pope410211   |
+| pcreglow     | pcreglow     |
+| melrodbos    | MStaehling   |
+| pope410211   | JessyRiordan |
+| kmattiko     | MStaehling   |
+| gatorpazz    | pope410211   |
+
+### Following the Blueprints
+
+Merge your `05` branch via the Big Green Button, update your local master, and start a _new_ branch called `06--temple-of-dom` inside your `TIY-Assignments` repo. First order of business: clean house. Delete the contents of `scripts.js` in all three Blusprints folders and commit the deletion. Push the branch and open your PR to get to your first checkpoint.
+
+Next, get your `NestedAccordion` working with just _one_ accordion in _one_ direction: opening when initially closed or closed when initially open. Then you can start to work on the toggle for that accordion. Finally, get the open and close working for _all_ accordions on the page, regardless of nesting level. Remember that the initial state should be _closed_... and for bonus points, make a "real accordion": one that only opens one panel at a time.
+
+After that, get to work on the `FullWidthTabs` interface, starting with the tabs. Making them respond to clicks _at all_ (tracer bullet anyone?) would be a good Baby Step. Getting them to activate on click will get you to the first checkpoint here. Can you get the panels to reveal as well?
+
+#### BEAST MODE
+
+Thought I forgot about our friend `QuotesRotator`...? Well, this one is a bit trickier. You'll need to learn something about the [`WindowTimers` API](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers) for that one.
+
+#### NIGHTMARE MODE
+
+That pretty progress bar won't animate itself, y'know... Or will it? Yeah, probably not, but remember to check out the CSS for clues! Find something you don't recognize? [Documentation for CSS is in the MDN, too!](https://developer.mozilla.org/en-US/docs/Web/CSS). Wonder if _that_ will come up next week?
+
```

Comment:
In retrospect, maybe `FullWidthTabs` isn't the right next step from `NestedAccordion`. While the concepts are similar -- when an element is clicked, toggle a class from another, related element -- the complexity os much higher. A better out-of-class assignment would be one that requires _an exact duplication_ of the `NestedAccordion` concepts in a _slightly different_ context. `FullWidthTabs` might be a good advanced assignment.